h5p-bildetema/src/components/HeaderLink/HeaderLink.module.scss

```diff
@@ -29,7 +29,7 @@
 }
 
 .headerLinkLabel {
-  @include breakpoint-max($small) {
+  @include breakpoint-max($medium) {
```

Comment:
Maybe we should do the same to the language selector also, so that the labels are hidden at the same breakpoint

Reply:
Yes